charts/kamus/templates/_helpers.tpl

```diff
@@ -33,4 +33,9 @@ KeyManagement__GoogleKms__KeyRingName: {{ .Values.keyManagement.googleKms.keyRin
 KeyManagement__GoogleKms__ProtectionLevel: {{ default "HSM" .Values.keyManagement.googleKms.protectionLevel }}
 KeyManagement__GoogleKms__CredentialsPath:  "/home/dotnet/app/secrets/googlecloudcredentials.json"
 {{ end }}
+{{ if .Values.keyManagement.awsKms }}
+KeyManagement__AwsKms__Region: {{ default "" .Values.keyManagement.googleKms.region }}
+KeyManagement__AwsKms__Key: {{ default "" .Values.keyManagement.googleKms.key }}
+KeyManagement__AwsKms__Secret: {{ default "" .Values.keyManagement.googleKms.secret }}
```

Comment:
This should go to the secret, not to the configuration that ends up in a config map.
Also, you're pointing to googleKms in the values :)
I think there is no need for default here, all the three are required